wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/devfile/DevfileParser.java

```diff
@@ -87,36 +88,54 @@ public DevfileParser(
    */
   public DevfileImpl parseYaml(String devfileContent) throws DevfileFormatException {
     try {
-      return parse(devfileContent, yamlMapper, emptyMap());
+      return parse(parseYamlRaw(devfileContent), yamlMapper, emptyMap());
     } catch (OverrideParameterException e) {
       // should never happen as we send empty overrides map
       throw new RuntimeException(e.getMessage());
     }
   }
 
   /**
-   * Creates {@link DevfileImpl} from given devfile content in YAML and provides possibility to
-   * override its values using key-value map, where key is an json-pointer-like string and value is
-   * desired property value. NOTE: unlike json pointers, objects in arrays should be pointed by
-   * their names, not by index. Examples:
```

Comment:
imo that comment about override rules is still useful; may we keep it ?

Reply:
added to `parseJsonNode` method